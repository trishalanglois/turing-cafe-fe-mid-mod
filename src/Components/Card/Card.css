article {
  border: 1px solid black;
  margin: .5vw;
  background-color: white;
  width: 300px;
  height: 200px;
}
