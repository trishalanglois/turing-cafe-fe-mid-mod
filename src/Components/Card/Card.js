import React from 'react';
import './Card.css';

const Card = ({ id, name, date, time, number, removeReservation }) => {
  return (
    <article>
      <h3>{name}'s Reservation</h3>
      <p>Date: {date}</p>
      <p>Time: {time}</p>
      <p>Number: {number}</p>
      <button className="cancel-button">Cancel</button>
    </article>
  )
}

export default Card;
