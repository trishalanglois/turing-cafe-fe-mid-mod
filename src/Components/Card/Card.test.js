import Card from './Card';
import React from 'react';
import { shallow } from 'enzyme';

describe('Card', () => {
  it('should match the snapshot with all data passed correctly', () => {
    const wrapper = shallow(<Card
      id={7}
      name='Trisha'
      date='10/22'
      time='11:00'
      number={9}
    />);

    expect(wrapper).toMatchSnapshot();
  })
})
