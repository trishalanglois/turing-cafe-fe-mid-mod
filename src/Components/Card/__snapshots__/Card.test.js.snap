// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot with all data passed correctly 1`] = `
<article>
  <h3>
    Trisha
    's Reservation
  </h3>
  <p>
    Date: 
    10/22
  </p>
  <p>
    Time: 
    11:00
  </p>
  <p>
    Number: 
    9
  </p>
  <button
    className="cancel-button"
  >
    Cancel
  </button>
</article>
`;
